lib/ansible/modules/source_control/git_config.py

```diff
@@ -64,6 +64,12 @@
         set that setting to the given value.
     required: false
     default: null
+  state:
+    description:
+      - Specify if a config should be C(set) or C(unset).
+    required: false
```

Comment:
Add a `choices` key with `present` and `absent`.

Reply:
The `choices` key has been added.